src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -556,6 +561,12 @@ export class CalciteCombobox {
 
   emitCalciteLookupChange = debounce(this.internalCalciteLookupChangeEvent, 0);
 
+  internalComboboxChangeEvent = (): void => {
+    this.calciteComboboxChange.emit(this.selectedItems);
```

Comment:
Good question. I didn't touch that because I wasn't sure. Is that the preferred way?

Reply:
I'll update it to have {selectedItems}

Reply:
I'd like it to be. 😄  I thought we had added a note on this on the conventions, but no dice. 

I think this would be good in the long run since it wouldn't be a breaking change if we needed to add another prop to the event payload. This also makes it simple for users where they don't need to distinguish between detail being the value itself or a value object.